src/api/api.ts

```diff
@@ -203,7 +203,23 @@ export class TodoistApi {
   }
 
   private async getSections(): Promise<Result<ISectionRaw[], Error>> {
-    const url = `https://api.todoist.com/rest/v1/sections`;
+    // For some reason the sections do now follow the string-based IDs rule
+    // We deal with this in an isolated centralized way here, so that we can keep the rest of the
+    // code homogeneous, using string-based section IDs all around.
+    type SectionRawFromAPI = Omit<ISectionRaw, "id" | "project_id"> & {
+      id: number;
+      project_id: number;
+    };
```

Comment:
If we expect this to change sometime in the future, we should future proof this such that when the JSON is changed, the plugin doesn't break.

I imagine doing `number | string` would be sufficient here to handle both cases

Reply:
Actually it looks as though the API for this has been fixed. At least, when I tested it manually it was giving me `string`s back.